src/main/java/com/hm/achievement/listener/statistics/AbstractListener.java

```diff
@@ -103,4 +110,13 @@ int getInventoryAvailableSpace(Player player, ItemStack newItemStack) {
 
 		return availableSpace;
 	}
+
+	Set<String> findAdvancementsByCategoryAndName(MultipleAchievements category, String identifier) {
```

Comment:
I believe this whole method body could be simplified to the following:
```
return mainConfig.getShallowKeys(category.toString()).stream()
				.filter(keys -> ArrayUtils.contains(StringUtils.split(keys, '|'), identifier))
				.collect(Collectors.toSet());
```
The `getShallowKeys` helper handles non existing configuration sections and is a shorthand for the `getConfigurationSection(category.toString()).getKeys(false)` chaining. 

Could you also please add a comment briefly explaining what the method does, and clean the unused imports in this class? 😃